Tensile/Source/lib/source/hip/HipSolutionAdapter.cpp

```diff
@@ -35,12 +35,6 @@
 #include <Tensile/hip/HipSolutionAdapter.hpp>
 #include <Tensile/hip/HipUtils.hpp>
 
-//@TODO add alternative for windows
```

Comment:
Why is this part of code removed?

Reply:
These includes are never used in this file, so this is just unnecessary.